ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -194,208 +319,237 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
 
       return <div style={{marginBottom: 40}}>
         <div style={styles.inputLabel}>{title}</div>
-        {isLong ?
-          <TextArea
+        {isLong ? <TextArea  data-test-id={props.id || valueKey}
             style={styles.longInputStyle}
             {...inputProps}
-          /> :
-          <TextInput
+          />  :
+            <TooltipTrigger content='This field cannot be edited' disabled={!props.disabled}>
+          <TextInput  data-test-id={props.id || valueKey}
             style={styles.inputStyle}
             {...inputProps}
-          />}
+          /></TooltipTrigger>}
         <ValidationError>{errorText}</ValidationError>
       </div>;
     };
 
-    const removeOldInstitutionalAffiliation = (affiliation: InstitutionalAffiliation) => {
-      this.setState(fp.update(
-        ['currentProfile', 'institutionalAffiliations'],
-        fp.pull(affiliation)
-      ));
-    };
 
-    const addEmptyOldInstitutionalAffiliation = () => {
-      this.setState(fp.update(
-        ['currentProfile', 'institutionalAffiliations'],
-        affiliation => fp.concat(affiliation, {institution: '', role: ''})));
-    };
-
-    const renderVerifiedInstitutionalAffiliationComponents = () => {
-      const {verifiedInstitutionalAffiliation} = profile;
-      if (!verifiedInstitutionalAffiliation) {
-        return;
-      }
-
-      const {institutionDisplayName, institutionalRoleEnum, institutionalRoleOtherText} = verifiedInstitutionalAffiliation;
-      return <React.Fragment>
-        <div style={{...styles.h1, marginBottom: 24}}>
-          Verified Institutional Affiliation
-        </div>
-        <FlexRow>
-          <FlexColumn>
-            <div style={styles.inputLabel}>
-              Institution
-            </div>
-            <div style={styles.uneditableProfileElement}>{institutionDisplayName}</div>
-          </FlexColumn>
-          <FlexColumn>
-            <div style={styles.inputLabel}>
-              Role
-            </div>
-            <div style={styles.uneditableProfileElement}>
-            {institutionalRoleEnum === InstitutionalRole.OTHER ? institutionalRoleOtherText : institutionalRoleEnum}
-            </div>
-          </FlexColumn>
-        </FlexRow>
-      </React.Fragment>;
-    };
+    return <FadeBox style={styles.fadebox}>
+      <div style={{width: '95%'}}>
```

Comment:
Everything within this div should be indented once more.

Reply:
In general, I'd like to see everything within an open tag indented once, and de-indented after the closed tag, so that all elements at a given level of the heirarchy are at the same level of indentation.